packages/dds/tree/src/feature-libraries/chunked-forest/codec/format.ts

```diff
@@ -64,6 +64,12 @@ export const EncodedFieldShape = Type.Tuple([
 
 export type EncodedFieldShape = Static<typeof EncodedFieldShape>;
 
+/**
+ * Content of the encoded field is an array of chunk reference ids, one for each chunk. Each chunk in the field is
+ * encoded separately and is referenced by its reference id.
```

Comment:
This is a great comment, though I think linking a type for `chunk reference ids` would make it even better.

Also a minor style note: when working with markdown like text which will be re-wrapped when presented to users (in this case in IntelliSense and our API docs), I think it cleaner to wrap the lines at sentence boundaries when practical. That makes line based tools like git and our code review comments less conflict prone, and more semantic.